modules/upgrade/pages/migrate/kubernetes/strimzi.adoc

```diff
@@ -0,0 +1,351 @@
+
+CRDs act as configuration instructions to describe the custom resources in a Kubernetes cluster.
+CRDs also allow resources to benefit from native Kubernetes features like CLI accessibility and configuration validation.
+
+Both Strimzi and the Redpanda Operator offer CRDs for creating and managing components as Kubernetes resources. Some custom resources in Strimzi do not have an equivalent custom resource in Redpanda.
+
+[cols="3", options="header"]
+|===
+| Strimzi custom resource | Redpanda equivalent | Notes
+
+| Kafka
+| Redpanda custom resource
+| See <<Migrate the Kafka resource>>.
+
+| KafkaBridge
+| Built-in HTTP proxy
+| See <<Migrate the KafkaBridge resource>>.
+
+| KafkaConnect
+| Available as a Helm chart
+| See <<Migrate the KafkaConnect resource>>.
+
+| KafkaMirrorMaker2
+| Included in the Connectors Helm chart
+| See <<Migrate the KafkaMirrorMaker2 resource>>.
+
+| KafkaNodePool
+| No support for heterogeneous brokers
+| See <<Migrate the KafkaNodePool resource>>.
+
+| KafkaTopic
+| Topic custom resource
+| See <<Migrate the KafkaTopic resource>>.
+
+| KafkaUser
+| Users are managed through cluster configuration
+| See <<Migrate the KafkaUser resource>>.
+
+|===
+
+== Migrate to Redpanda Operator
+
+To ensure a smooth migration:
+
+* <<Plan data and topic migration>>.
+* <<Migrate Strimzi resources to Redpanda>>.
+* <<Adjust monitoring and alerting>>.
+* <<Engage with the Redpanda community>>.
+
+=== Plan data and topic migration
+
+When migrating from Strimzi to Redpanda Operator, plan your data and topic migration to ensure data integrity and minimal downtime:
+
+- *Back up data*: Before starting the migration, ensure that all data in Kafka topics is backed up.
+
+- *Migrate topics*: Review the configuration of your existing Kafka topics, including partition count, replication factor, and any custom configurations like retention policies. This information is essential for migrating these topics to Redpanda. See <<Migrate the KafkaTopic resource>>.
+
+- *Transfer data*: Determine the most suitable method for data transfer. For example, you can use MirrorMaker2 to replicate data from Kafka to Redpanda in real-time. This method is useful if you need to keep the source system online during the migration. See <<Migrate the KafkaMirrorMaker2 resource>>.
+
+- *Connect clients*: Connect client applications to Redpanda.
+
+=== Migrate Strimzi resources to Redpanda
+
+Migrating from Strimzi to Redpanda involves converting Strimzi custom resources into their corresponding forms for the Redpanda Operator. This process ensures that your Kafka configurations and setups are correctly translated and optimized for Redpanda.
+
+These example Strimzi manifests are for version {strimzi-link}[{strimzi-version}] of the Strimzi Operator.
+
+==== Migrate the Kafka resource
+
+This section provides an example of how to translate configuration from a Strimzi Kafka resource to a Redpanda resource.
+
+[.side-by-side]
```

Comment:
Fantastic use of this features--loved being able to go through the numbers in the code block and read along about the behavior on both sides. Awesome!